### ISSUE TITLE
[Task]: only sync AMO specific policies from cinder

### ISSUE THREAD

**Issue by eviljeff** [mozilla/addons#14791]
### Description

Implements https://mozilla-hub.atlassian.net/browse/AMO2-671

Cinder’s API now returns labels with policies   which should enable us to selectively add only “AMO” labelled policies to our database as CinderPolicy instances

```json
{
        "last_updated_by": {
          "id": 43,
          "email": "x@mozilla.com"
        },
        "created_by": {
          "id": 43,
          "email": "x@mozilla.com"
        },
        "status": {
          "id": 2,
          "name": "PUBLISHED"
        },
        "uuid": "97b09ece-f949-4a82-9405-963b024d6ce6",
        "description": "This content violates the law in a jurisdiction where Mozilla content is displayed, which is not permitted under Mozilla's Acceptable Use Policy. (See < >.)\n\n",
        "nested_policies": [],
        "labels": [
          {
            "name": "AMO",
            "description": "AMO"
          }
        ],
        "name": "Illegal content ",
        "created_at": "2024-01-23T16:38:26.539Z",
        "updated_at": "2024-05-02T03:10:19.421Z"
      },
```

### Acceptance Criteria

```[tasklist]
  ### Acceptance Criteria
  - [ ] executing the policy sync task only syncs policies that are labelled with "AMO"
  ```

### Checks

- [X] If the issue is ready to work on, I have removed the "needs:info" label.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/AMO2-767)

**Comment by ioanarusiczki** [comment]
I see in Cinder - [Policies](https://stage.cinder.nonprod.webservices.mozgcp.net/policies?policyFilters=%7B%7D) that the list has policies labeled with "AMO" or "MoSo". 

I assume I should see in [admin ](https://addons-internal.dev.mozaws.net/en-US/admin/models/abuse/cinderpolicy/?all=&o=0.3) only what's "AMO" 

"Approve" and "Copyright and trademark" have both labels , should also be available after sync.

**Comment by eviljeff** [comment]
@ioanarusiczki yes - policies labelled with AMO or no label. And it should also remove any policies that are labelled with other labels, if they aren't connected to  ReviewActionReason or a decision

**Comment by ioanarusiczki** [comment]
After sync the number of policies dropped from 139 to 87. 

What's with AMO label in Cinder is also present in AMO admin including: Approve, Ignore, Escalate policies + their sub-policies. 

There's one sub-policy in admin , id 37 "Online gambling" from "Acceptable Use" which is not present in [Cinder](https://stage.cinder.nonprod.webservices.mozgcp.net/policies/0d9df565-f249-40f8-8954-e73e65932ca2).
That's something I've noticed missing in Cinder in the action list for reports and I filed it on [Lindie](https://lindie.app/share/6a21d831b39351d7c6fe898f6d22619af62dde98/PROD-1529)
